legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/store/routing.pure

```diff
@@ -230,7 +230,9 @@ function  meta::pure::router::store::routing::specializedFunctionExpressionRoute
                                                | getRoutingStrategyFromMappingAndRuntime($fromMapping, $fromRuntime));
                 let newState               = ^$state(routingStrategy = $newRoutingStrategy);
 
-                let processedFunction      = routeFunctionExpressionFunctionDefinition($fe.parametersValues->at(0)->cast(@FunctionExpression).func, $fe.parametersValues->at(0)->cast(@FunctionExpression), $newState, $fromExecutionContext, $vars, $inScopeVars, $extensions, $debug);
+                // todo check why this fix my problem
```

Comment:
Need to check with @hardikmaheshwari @gs-ssh16 why we changes this last week as part of preval.